### PR TITLE
New package: light-0.9

### PR DESCRIPTION
**Pull request by d-kraus**

### CODE REVIEW

File: srcpkgs/light/INSTALL [hunk 1]
```diff
@@ -0,0 +1,6 @@
+case "${ACTION}" in
+post)
+	sudo chown root:root /usr/bin/light 
+	sudo chmod 4577 /usr/bin/light
```
Comment (chneukirchen) [inline srcpkgs/light/INSTALL]: This sounds wrong.

File: srcpkgs/light/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'light'
+pkgname=light
+version=0.9
+revision=1
+build_style=gnu-makefile
+short_desc="Light is a program to control backlight controllers under GNU/Linux"
```
Comment (chneukirchen) [inline srcpkgs/light/template]: Drop "Light is a"

Comment (chneukirchen) [inline srcpkgs/light/template]: and "under GNU/Linux"

File: srcpkgs/light/INSTALL (hunk 1 above)
Comment (Vaelatern) [inline srcpkgs/light/INSTALL]: The use of sudo us wrong here every time, the paths must all be relative dirs not absolute.

File: srcpkgs/light/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+maintainer="Dennis Kraus <kraus@posteo.de>"
+license="GPL-3"
+homepage="http://haikarainen.github.io/light/"
+distfiles="https://github.com/haikarainen/light/archive/v${version}.tar.gz"
+checksum=a6015bad4d2b17507a140896d7f30b4ab47eb7a0faef670edf3f1e30de9cdfb6
+
+
+do_build() {
+	echo $CFLAGS
+	$CC $CFLAGS --std=c89 -O2 -pedantic -Wall -I "./include" -g -o light src/helpers.c src/light.c src/main.c $LDFLAGS
```
Comment (chneukirchen) [inline srcpkgs/light/template]: use *.c perhaps. Or patch the makefile to use LDFLAGS.

File: srcpkgs/light/template [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+short_desc="Light is a program to control backlight controllers under GNU/Linux"
+maintainer="Dennis Kraus <kraus@posteo.de>"
+license="GPL-3"
+homepage="http://haikarainen.github.io/light/"
+distfiles="https://github.com/haikarainen/light/archive/v${version}.tar.gz"
+checksum=a6015bad4d2b17507a140896d7f30b4ab47eb7a0faef670edf3f1e30de9cdfb6
+
+
+do_build() {
+	echo $CFLAGS
```
Comment (chneukirchen) [inline srcpkgs/light/template]: Drop this.

**Comment by d-kraus** [comment]
Hey,
i hope i addressed all issues with the new version.

File: srcpkgs/light/template [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+distfiles="https://github.com/haikarainen/light/archive/v${version}.tar.gz"
+checksum=a6015bad4d2b17507a140896d7f30b4ab47eb7a0faef670edf3f1e30de9cdfb6
+
+pre_build() {
+	sed -i 's|^CFLAGS=|override CFLAGS+=|g' Makefile
+	sed -i 's|src/main.c$|src/main.c $(LDFLAGS)|' Makefile
+}
+
+do_install() {
+	install -Dm 4755 light $DESTDIR/usr/bin/light
```
Comment (chneukirchen) [inline srcpkgs/light/template]: Use vbin

Comment (d-kraus) [inline srcpkgs/light/template]: According  to the documentation vbin sets the permission 0755.
But light requires setuid. Therefor i changed the template to utilize vinstall.

**Comment by d-kraus** [comment]
push

**Comment by Vaelatern** [comment]
Would have prefered a single commit. Squashing thanks to github, but it's easier to take just one commit.

**Pull request merged**